### ISSUE TITLE
error: Error loading documents - api response: {"method":"GET","status":400,"response":{"error":"v1_retired"}}

### ISSUE THREAD

**Issue by nt1m** [PalmerAL/DocLayer#215]
Looks like this uses an old version of the dropbox api

**Comment by PalmerAL** [comment]
Yeah, it hasn't worked for a few months now, and I'm not working on it anymore. I guess I'll add a notification to the website, or if you're interested in continuing to work on this I can add you to the repository.
